server/utils/assessments/utils.ts

```diff
@@ -284,6 +284,9 @@ const getSectionSuffix = (task: Task) => {
   return `<p><a href="${link}">${copy}</a></p>`
 }
 
+const decisionFromAssessment = (assessment: Assessment) =>
+  assessment?.data?.['make-a-decision']?.['make-a-decision']?.decision || ''
```

Comment:
Not for now, but I wonder if it might be less brittle if we get the response from the page objects themselves somehow, rather than walking the data in this way (as we're doing it a lot now!)